pkg/hintrunner/zero/zerohint_signatures_test.go

```diff
@@ -26,6 +28,126 @@ func TestSignatures(t *testing.T) {
 				},
 			},
 		},
+		"GetPointFromX": {
+			{
+				//> if v % 2 == y % 2
```

Comment:
nitpick: tab space doesn't match with the other comments

Reply:
refactored the comments